nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
I explained in the commit b9f6dfe8c5, but I see I should add a source-code comment.

The consequence is that even the manual evaluation does *not* depend on those packages. You can check `$ nix-store -q --references "$(nix-instantiate nixos/release.nix -A manual)"`.